src/telemetry.ts

```diff
@@ -0,0 +1,38 @@
+import TelemetryReporter from 'vscode-extension-telemetry';
+import { BaseLanguageClient, ClientCapabilities, StaticFeature } from 'vscode-languageclient';
+
+const TELEMETRY_VERSION = 1;
+
+type TelemetryEvent = {
+  v: number;
+  name: string;
+  properties: { string };
```

Comment:
To avoid an implicit any here, we should specify a type for the object values, too.
```suggestion
  properties: { [key: string]: unknown };
```